internal/server/server.go

```
@@ -59,6 +59,12 @@ func createSOMASServer(
 		forageHistory[t] = make([]foraging.ForagingReport, 0)
 	}
 
+	if gameConfig.IIGOConfig.StartWithRulesInPlay {
+		for ruleName := range rules.AvailableRules {
+			_ = rules.PullRuleIntoPlay(ruleName)
```

can you add a comment as to why this err is ignored?

---

Yup, thanks

---

I personally still wouldn't ignore btw--at least log it out in case a bug happens so it's easier to debug